src/services/resource.js

```diff
@@ -210,6 +327,394 @@ module.exports = class resourceHelper {
+						user_id,
+						finalResourceIds
+					)
+				}
+				// if the organization have any resource type in parallel review type
+				if (resourceTypesInParallelReview.length > 0) {
+					// fetch all parallel resource ids from org which is open to all
+					const parallelResourcesIds = await this.findParallelResources(
+						organization_id,
+						resourceTypesInParallelReview
+					)
+					// add the resource ids in the final array
+					finalResourceIds = [...finalResourceIds, ...parallelResourcesIds]
+				}
+
+				const assignedToMe = await this.findResourcesAssignedToReviewer(uniqueOrganizationIds, user_id)
+
+				finalResourceIds = [...finalResourceIds, ...assignedToMe]
+
+				finalResourceIds = _.difference(
+					utils.getUniqueElements(finalResourceIds),
+					utils.getUniqueElements(resourceIdsToBeRemoved)
+				)
+			}
+
+			if (finalResourceIds.length === 0) {
+				return responses.successResponse({
+					statusCode: httpStatusCode.ok,
+					message: 'RESOURCE_LISTED_SUCCESSFULLY',
+					result: [],
+				})
+			}
+
+			const resourceFilter = {
+				id: { [Op.in]: finalResourceIds },
+				organization_id: { [Op.in]: uniqueOrganizationIds },
+				user_id: {
+					[Op.notIn]: [user_id],
+				},
+			}
+			if (searchText != '')
+				resourceFilter.title = {
+					[Op.iLike]: '%' + searchText + '%',
+				}
+
+			if (common.TYPE in queryParams && queryParams[common.TYPE]) {
+				resourceFilter.type = queryParams[common.TYPE].split(',')
+			}
+
+			const response = await resourceQueries.resourceList(
+				resourceFilter,
+				[
+					'id',
+					'title',
+					'type',
+					'organization_id',
+					'status',
+					'user_id',
+					'submitted_on',
+					'last_reviewed_on',
+					'created_at',
+					'meta',
+				],
+				sort,
+				page,
+				limit
+			)
+
+			if (response.length === 0) {
+				return responses.successResponse({
+					statusCode: httpStatusCode.ok,
+					message: 'RESOURCE_LISTED_SUCCESSFULLY',
+					result: [],
+				})
+			}
+
+			const uniqueCreatorIds = utils.getUniqueElements(
+				response.result.map((item) => {
+					return item.user_id
+				})
+			)
+
+			const userDetails = await this.fetchUserDetails(uniqueCreatorIds)
+			const orgDetails = await this.fetchOrganizationDetails(uniqueOrganizationIds)
+
+			result.data = response.result.map((item) => {
+				let returnValue = item
+
+				if (item.meta?.notes) returnValue.notes = item.meta.notes
+				if (inProgressResources.includes(item.id)) {
+					returnValue.reviewer_status = common.REVIEW_STATUS_INPROGRESS
+				}
+				returnValue.creator =
+					userDetails[item.user_id] && userDetails[item.user_id].name ? userDetails[item.user_id].name : ''
+				returnValue.organization = orgDetails[item.organization_id]
+				delete item.user_id
+				delete item.organization_id
+				delete returnValue.meta
+				return returnValue
+			})
+			result.count = response.count
+			result.in_progress_count = in_progress_count
```

Comment:
@adithyadinesh0412 whenever you pass empty result then if inprogress count is required then add that